The-SMIRNOFF-force-field-format.md

```diff
@@ -15,6 +31,8 @@ The file has tags corresponding to OpenMM force terms (`BondForce`, `AngleForce`
 ```
 which introduces following `Angle` terms which will use units of degrees for the angle and kilocalories per mole per square radian for the force constant.
 
+**QUESTION:** Can we change `smirks` to `smarts` in the attributes, since these are really SMARTS strings augmented with atom tagging (which seems to be broadly supported)?
```

Comment:
In a way it feels like that ship has sailed when we went with "SMIRNOFF"; it's semantics that isn't particularly important. (e.g. I haven't seen a SMARTS spec that HAS the tags yet, even though they are supported in many cases (though there may be such a spec), and it's simple enough to just say that for our purposes, we use SMIRKS as SMARTS with tags. Otherwise you'll have to explain (more often) why it's SMIRNOFF and not SMARNOFF.

Reply:
What David says; we might want to point out that the only element of SMIRKS that we use is the atom annotation. Yes this is a broadly supported extension, but it is still SMIRKS.